File: pl-domain/municipality-service/src/main/java/com/polylab/municipality/domain/service/MunicipalityServiceImpl.java
package com.polylab.municipality.domain.service;

import com.jxareas.jxcore.domain.service.PersistentService;
import com.polylab.municipality.persistence.model.Municipality;
import com.polylab.municipality.persistence.repository.MunicipalityRepository;
import org.springframework.stereotype.Service;

@Service
public class MunicipalityServiceImpl extends PersistentService<Municipality, Integer> implements MunicipalityService {
    public MunicipalityServiceImpl(MunicipalityRepository repository) {
        super(repository);
    }
}

File: pl-domain/municipality-service/src/main/java/com/polylab/municipality/domain/service/MunicipalityServiceImpl.kt
package com.polylab.municipality.domain.service

import com.jxareas.efficax.domain.service.PersistentService
import com.polylab.municipality.persistence.model.Municipality
import com.polylab.municipality.persistence.repository.MunicipalityRepository
import org.springframework.stereotype.Service

@Service
class MunicipalityServiceImpl(repository: MunicipalityRepository) :
    PersistentService<Municipality, Int>(repository), MunicipalityService
